### PR TITLE
Mat 3609 measure rationale

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-3609](https://jira.cms.gov/browse/MAT-3609)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/measureDetails/MeasureDetails.test.tsx [hunk 1]
```diff
@@ -43,6 +46,10 @@ MeasureDisclaimerMock.mockImplementation(() => {
   return <div>Mock Measure Disclaimer</div>;
 });
 
+MeasureRationaleMock.mockImplementation(() => {
+  return <div>Mock Measure Rationale</div>;
+});
+
```
Comment (RohitKandimalla) [inline src/components/editMeasure/measureDetails/MeasureDetails.test.tsx:52]: jest.mock"./measureRationale/MeasureRationale", () => () => (
  <div>Mock Measure Rationale</div>
));

Wondering if we can mock and return an element immediately. The only advantage I am seeing is that we could reduce number of lines of code.

File: src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    measureServiceApi
+      .updateMeasure(measure)
+      .then(() => {
+        setSuccess(true);
+      })
+      .catch((reason) => {
+        const message = `Error updating measure "${measure.measureName}"`;
+        console.error(message);
+        console.error(reason);
+        setError(message);
```
Comment (RohitKandimalla) [inline src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx:48]: I believe, We don't have to log errors to console. Those are visible only to user and are useless.

File: src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      });
+  };
+
+  return (
+    <Form onSubmit={onFormSubmit} data-testid="measureRationale">
```
Comment (RohitKandimalla) [inline src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx]: Why Formik is not being used ? unnecessary ?

File: src/components/editMeasure/measureDetails/MeasureDetails.test.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/MeasureDetails.test.tsx:52]: Thanks Rohit. I tried it, didn't work, I get "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object." error.

File: src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx:48]: I totally agree! Removed it.

File: src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx (hunk 3 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx]: I guess I can.

Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureRationale/MeasureRationale.tsx]: @RohitKandimalla I just pushed in the changes for using Formik, can you please help have a look? Thanks so much!

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**